hubblestack/modules/fdg.py

```diff
@@ -247,7 +262,15 @@ def top(fdg_topfile='top.fdg'):
     (optional) ``starting_chained`` value dumped to a string. The values
     in the dictionary are the associated returns from the fdg runs.
     """
-    fdg_routines = _get_top_data(fdg_topfile)
+    fdg_routines = _get_top_data(fdg_topfile, fdg_version)
+    # there are currently two cases for fdg_version 'fdg' and 'fdg_v2'. 
+    # We call the function 'fdg' when the old fdg is refered.
+    # And 'run' when fdg_v2 is called.
+    if fdg_version == 'fdg':
+        call_function = fdg
+    else:
```

Comment:
As we have done input check at other places. We can check this value can be only in (fdg, fdg_v2)

Else here, for any value other than 'fdg', it will run v2

Reply:
done